block-storage-attacher/tests/e2e/e2e.sh

```diff
@@ -9,6 +9,13 @@ set +a
 # check mandatory variables
 [ -z "$GOPATH" ] && echo "Need GOPATH for plugin build and test executions(e.g export GOPATH=\path\to)" && exit 1
 
+E2E_PATH="src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/e2e-tests/"
```

Comment:
This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.